README.md

```diff
@@ -311,7 +311,8 @@ There is a JSBin demo for each of the binding handlers:
 - [`init` with value binding](http://jsbin.com/xuluye/)
 - [`init` with html binding](http://jsbin.com/wilahi/)
 - [`init` with attr binding](http://jsbin.com/zehivi/)
-- [`init` with checked binding](http://jsbin.com/zemode/)
+- [`init` with checked binding for checkbox](http://jsbin.com/zemode/)
+- [`init` with checked binding for radio](http://jsbin.com/pumodetaqu/)
```

Comment:
I have one suggestion: the UI currently displays "Plays" before each radiobutton. Could you add what is being played to the text?